user/gitoxide/template.py

```diff
@@ -13,13 +13,12 @@
     "curl-devel",
     "rust-std",
     "sqlite-devel",
-    "zlib-ng-compat-devel",
```

Comment:
They started using `zlib-rs` in https://github.com/GitoxideLabs/gitoxide/pull/1963, meaning this is being replaced with a pure rust implementation